app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
+      Array.wrap(self[Solrizer.solr_name('title')])
+    end
+
+    def all_descriptions
```

Comment:
Well...we'll need to do this for collections, too. Right. Collections have multiple titles and descriptions?